packages/fp-felles/src/navAnsatt/access.jsx

```diff
@@ -29,9 +29,11 @@ const accessSelector = (validNavAnsattPredicates, validFagsakStatuses, validBeha
   return { employeeHasAccess, isEnabled };
 };
 
-export const writeAccess = accessSelector(
+export const writeAccess = behandlingType => accessSelector(
   [kanSaksbehandle],
-  [fagsakStatusCode.OPPRETTET, fagsakStatusCode.UNDER_BEHANDLING],
+  behandlingType.kode === BehandlingType.KLAGE
```

Comment:
Hvilket problem løser dette? Ser ut som dette styrer skrivetilgang for SB rimelig overordnet. Trenger dette å løses unikt for klage, og på dette nivået?

Reply:
@tor-nav ☝️

Reply:
Eventuelt, hva påvirker dette på andre områder enn klage?

Reply:
Det eneste alternativet som lar seg gjennomføre på kort sikt er å fjerne hele sjekken på fagsakstatus. Men da vil man ha skrivetilgang uansett behandlingstype. Jeg vet ikke om det ødelegger noe, men jeg vil jo tro at det er en grunn til at denne sjekken er lagt inn i utgangspunktet.

Reply:
har svart via slack